homeassistant/components/automation/state.py

```diff
@@ -58,8 +58,14 @@
     period: Dict[str, timedelta] = {}
 
     @callback
-    def state_automation_listener(entity, from_s, to_s):
+    def state_automation_listener(event: Event):
         """Listen for state changes and calls action."""
+        entity: str = event.data["entity_id"]
+        if entity not in entity_id:
+            return
```

Comment:
```suggestion
		if event.data["entity_id"] != entity_id:
		    return
```

Reply:
That doesn't work, as `entity_id = config.get(CONF_ENTITY_ID)` is a list of entities (`cv.entity_ids`)